.github/workflows/release.yml

```diff
@@ -26,4 +26,5 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          
```

Comment:
Thanks for applying the changes. However and empty line should be empty, if you remove these spaces it would work.

Reply:
done 😃  YAML editor config 🙃